test/e2e/lib/auditlogs.go

```diff
@@ -60,7 +62,7 @@ func MakeAuditLogsJobOrDie(client *Client, methodName, project, resourceName, se
 		Value: resourceName,
 	}, {
 		Name:  "TYPE",
-		Value: v1alpha1.CloudAuditLogsSourceEvent,
+		Value: eventType,
```

Comment:
Yep.
For source->target and source->broker->ksvc, they are different,
one use `v1alpha1.CloudAuditLogsSourceEvent`, the other use `lib.E2EAuditLogsRespType`.
To avoid the bug that we bypass the broker and don't check the reply events, I changed the type of the reply events to `lib.E2EAuditLogsRespType`, you can check the logic here https://github.com/capri-xiyue/knative-gcp/blob/536bbadd255c3f1c9c805b6b0b62b410e8ca3251/test/test_images/auditlogs_receiver/main.go#L60